### ISSUE TITLE
Question about the loss computing

### ISSUE THREAD

**Issue by promisechen666** [tudelft/ssl_e2vid#6]
In the code loss/flow.py
line 35 
 “param event_list: [batch_size x N x 4] input events (y, x, ts, p) ”  
the input seems to be   (ts,y, x, p) rather than (y, x, ts, p)

line 44-46 
 “     flow_idx = event_list[:, :, 1:3].clone()
        flow_idx[:, :, 0] *= self.res[1]  # torch.view is row-major
        flow_idx = torch.sum(flow_idx, dim=2)
” 
does it mean flow_idx = 240*y+x ？ I don't know if I misunderstood
